x-pack/plugins/security_solution/common/experimental_features.ts

```diff
@@ -236,6 +236,11 @@ export const allowedExperimentalValues = Object.freeze({
    * Enables the siem migrations feature
    */
   siemMigrationsEnabled: false,
+
+  /**
+   * Enables the Defend Insights feature
+   */
+  defendInsights: false,
```

Comment:
Thanks for adding this. I searched around and I don't actually see it being used... Where is it checked?

Reply:
Each of the 3 API handlers check for it like [this](https://github.com/joeypoon/kibana/blob/eb9ab8319a6e3940d509ca2d26356ba75ba572a6/x-pack/plugins/elastic_assistant/server/routes/defend_insights/post_defend_insights.ts#L73). We don't actually use it in `security_solution` plugin yet but will for following PRs.

Reply:
@joeypoon - Thats strange (to me). So somehow this `elastic_assistant` plugin has access to our internal features? The code you pointed to is a bit hard to follow. Do you know how where it is actually is checking the feature flag is enabled/disabled?

Reply:
We're checking for it at the beginning of all 3 new API handlers (in elastic_assistant) and that definition for those is actually [here](https://github.com/joeypoon/kibana/blob/00596173de5e86acba862626522a7d3b4b015fcc/x-pack/packages/kbn-elastic-assistant-common/impl/capabilities/index.ts#L23). The definition in `x-pack/plugins/security_solution/common/experimental_features.ts` isn't actually being used right now and is just prep for next PRs to have less conflicts.

Reply:
yeah. ok. I'm just a bit confused how the FF from security solution will be used in another plugin.

Reply:
> yeah. ok. I'm just a bit confused how the FF from security solution will be used in another plugin.

It's not, it will be used within the security solution plugin. It's only introduced here since it's the first PR being merged for the overall feature.